pkg/network/netpoll/connection.go

```diff
@@ -49,6 +49,12 @@ func (c *Conn) Peek(n int) (b []byte, err error) {
 	return
 }
 
+func (c *Conn) Read(p []byte) (int, error) {
```

Comment:
What is the purpose of adding this method?

Reply:
To convert conn.Read's return value from netpoll.ErrEOF to io.EOF